test/ios/App-Info.plist

```diff
@@ -34,6 +36,8 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>MGLMapboxMetricsEnabledSettingShownInApp</key>
+	<true/>
```

Comment:
We should probably actually do this, as well as write tests against it as part of https://github.com/mapbox/mapbox-gl-native/issues/1363#issuecomment-97195050.